packages/build/webpack/webpack.dev.config.js

```diff
@@ -29,16 +29,19 @@ module.exports = {
     filename: '[name].js',
     chunkFilename: '[name].js',
   },
-  devtool: false,
+  devtool: 'source-map',
```

Comment:
Btw, here's our [previous discussion about source maps](https://gravitational.slack.com/archives/C02DQ1C2BMW/p1651593622899069). From what I can gather, Alexey was against enabling source maps by default. Nowadays I think most of us would rather have them enabled by default.

Perhaps we could add an env var here so that if someone wants to keep it set to `off` they could just set the env var?

Reply:
Sure thing, I'll add an env var to override it

Reply:
Here is some recollection from my memory :) 

Source maps were disabled because they added extra overhead to build and debugging time. They also were not very useful (chrome struggled at that time to provide quick variable lookups and on the fly editing) so we disabled the source maps completely. Also, once you disable ES2015 compilation target (in babel) you should be able to see transpiled code pretty much unchanged. 

As for webpack ts complication in dev mode, the reason why it was disabled because it was slow. And since vscode has built-in ts compiler (which runs by default) it seemed to work fine. The feedback loop was < 1 sec. These days when everyone has M1 it might not be an issue anymore :)